lang/pkl/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               java 1.0
+PortGroup               github 1.0
+
+github.setup            apple pkl 0.25.1
+
```

Comment:
please add `github.tarball_from archive` ; this will require recalculate/update the checksums

Reply:
please add `revision 0`